### ISSUE TITLE
Pagination is broken.

### ISSUE THREAD

**Issue by nkgokul** [mglaman/material_admin#21]
Currently the pagination is broken on most of the admin pages. 

![image](https://cloud.githubusercontent.com/assets/1375897/5602301/fc6971ae-9369-11e4-91a5-b050dbc5df4c.png)

**Comment by mglaman** [comment]
Correct, there's a parent issue in #6 on researching how to do pagination within Material Design.
